llvm.py

```diff
@@ -98,15 +95,14 @@ def clean(opts: BaseOpts, target: str):
 
     rm_rf(stamp_file)
 
-    make_args = [
-	    '-C', '%s/llvm' % opts.mono_source_root,
+    make_args = make_default_args(opts)
```

Comment:
Note: This is the only call which is actually a `clean` instead of a build. I used the default args there to factor out the verbose code - I *think* the `-j` won't hurt the clean logic, but I didn't test it specifically.

Reply:
Yes, I don't see why it would affect it.